### PR TITLE
update

### PR DESCRIPTION
**Pull request by MarcusNoble**
I did something AWESOME!

### CODE REVIEW

**Comment by MarcusNoble** [comment]
This fixed #1

**Comment by MarcusNoble** [comment]
Fixes #1

**Comment by MarcusNoble** [comment]
Closes #1

**Comment by MarcusNoble** [comment]
Resolves #1

File: README.md [hunk 1]
```diff
@@ -2,3 +2,5 @@ GitHubTests
 ===========
```
Comment (PeterParker) [inline README.md]: test

**Pull request merged**